api_docs/changelog.md

```diff
@@ -20,6 +20,14 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 244**
+* [`POST /register`](/api/register-queue): Added new boolean
+  field `realm_require_unique_names` as a part of setting to
+  enforce unique names in realm.
+* [`GET /events`](/api/get-events): Added new boolean field
+  `require_unique_names` as a part of setting to enforce
+  unique names in realm.
```

Comment:
Generally we list the endpoints on a single line; see for example, feature level 241.

(Yes, the fields have slightly different names, but it's just more concise).